packages/flame_test/test/flame_async_test.dart

```diff
@@ -11,24 +11,25 @@ void main() {
   testWithFlameGame(
     'runs all the async tests',
     (game) async {
-      await game.ensureAdd(Component());
+      final world = game.world;
+      await world.ensureAdd(Component());
```

Comment:
should we encourage people to always use the world even if they are not using the camera?

Reply:
I think that is probably a good way forward, most of the time they don't even have to extend FlameGame now, just the World. What do you think?

Reply:
Indeed